tests/Feature/Filament/PlaylistResourceTest.php

```
@@ -0,0 +1,291 @@
+
+        /* Act */
+        $response = Livewire::test(ListPlaylists::class)
+            ->callTableAction(DeleteAction::class, $playlist);
+
+        /* Assert */
+        $response->assertHasNoActionErrors();
+        $this->assertDatabaseMissing('playlists', [
+            'id' => $playlist->id,
+        ]);
+        $this->assertSoftDeleted($playlist);
+    }
```

PHPMD (2.15.0)

The method it_deletes_a_playlist is not named in camelCase. (undefined)

(CamelCaseMethodName)